packages/checkout/utils/validation/getValidityMessageForInput.ts

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
Other file names are hyphenated, is this one different for a particular reason?

Reply:
Thanks for pointing this out, @mikejolley. When moving the function `mustContain()` from index.ts into mustContain.ts, I've accidentally used camel case for the file name. I then used camel case for all files within the `packages/checkout/utils/validation/` folder. I just went ahead and changed all file names from camel case to kebab case.